test/torchaudio_unittest/transforms/transforms_test_impl.py

```diff
@@ -18,11 +18,11 @@ def _get_ratio(mat):
 
 
 class TransformsTestBase(TestBaseMixin):
-    def test_InverseMelScale(self):
```

Comment:
Since changing the test config, could you try make the threshold of this test tighter?

Reply:
I have tried it locally, but the performance is very close to before, will keep it as is.